modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js

```diff
@@ -454,7 +457,9 @@ class Form extends Component {
 	}
 
 	openSidebar() {
-		this.refs.sidebar.open();
+		if (!this.props.context.dataEngineSidebar) {
```

Comment:
`dataEngineSidebar` looks to me as an instance(?). Would be good to rename it as `isDataEngineSidebar`? 

/cc @matuzalemsteles